<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9ORLONM" failures="1" tests="12" name="Personal loan" time="0.23" errors="0" timestamp="2023-11-28T00:48:52 IST">
  <testcase classname="test.Day1" name="Demo" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day1.Demo(Day1.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.Day1" name="SecondTest" time="0.002"/>
  <testcase classname="test.Day2" name="loan" time="0.002"/>
  <testcase classname="test.Day3" name="MobileLoginCarLoan" time="0.007"/>
  <testcase classname="test.Day3" name="WebLoginCarLoan" time="0.009"/>
  <testcase classname="test.Day3" name="MobilesignoutcarLoan" time="0.013"/>
  <testcase classname="test.Day3" name="MobilesignoutcarLoan" time="0.011"/>
  <testcase classname="test.Day3" name="MobilesignoutcarLoan" time="0.003"/>
  <testcase classname="test.Day4" name="LoginAPIHomeLoan" time="0.003"/>
  <testcase classname="test.Day4" name="MobileLoginHomeLoan" time="0.001"/>
  <testcase classname="test.Day4" name="WebLoginHomeLoan" time="0.001"/>
  <testcase classname="test.Day3" name="LoginAPICarLoan" time="0.004"/>
</testsuite> <!-- Personal loan -->
